Demo/Viper-v2-Demo/Viper-v2-Demo/Modules/Details/PokemonDetailsViewController.swift

```diff
@@ -15,16 +15,80 @@ final class PokemonDetailsViewController: UIViewController {
     // MARK: - Public properties -
 
     var presenter: PokemonDetailsPresenterInterface!
+    
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var headerImageView: UIImageView!
 
     // MARK: - Lifecycle -
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        _setupView()
+        presenter.viewDidLoad()
+    }
+    
+    private func _setupView() {
```

Comment:
You can rename it to `setupTableView` and call if from `didSet` of the table view.